perllib/FixMyStreet/DB/Result/Contact.pm

```diff
@@ -204,4 +204,12 @@ sub category_uneditable {
     return 0;
 }
 
+sub add_note {
```

Comment:
Would be good to have a little t/app/mode/contact.t test testing this I guess.

Reply:
Added in 177284eb1fb26441aeaac12c0337e2656a2aaccf